sysutils/contacts/Portfile

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dhess contacts a4db4d86caa37553b8c4c6f70bd9d91bd34aa705
```

Comment:
This makes the version number `a4db4d86caa37553b8c4c6f70bd9d91bd34aa705` which is not what we want. Please use a release version or manually set the version to something sensible.

Reply:
The issue is, there is no release version containing this. As this project is also no longer maintained, I would have to create a pseudo-release myself, which I don't want to do. What to do? Just manually set the version to the last release version? Invent a version number? What is the best practice here?

Reply:
Perfect, thanks!